.changeset/itchy-comics-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ultraviolet/ui': minor
```

Comment:
reminder: if you add later a NumberInputFieldV2, you'll probably need to generate another changeset